### PR TITLE
Data 2

### PR DESCRIPTION
**Pull request by Jaxobc**
Changed the format of my comment section
Added delete functionality by adding a .java class
Waiting to deploy current version

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -85,6 +84,29 @@ <h2>Ballers</h2>
...
+          <input type="text" required pattern="\S(.*\S)?" maxlength="20" name="name"  textarea placeholder="Enter comment here..."><br>
+          <label for="comment">Comment:</label><br>
+          <input type="text" required pattern="\S(.*\S)?" maxlength="120" name="text-input" textarea placeholder="Enter comment here..."><br><br>
+          <label for="commentLimit">Choose the amount of comments you want to see:</label>
+          <select type="number" id="commentLimit" name="commentLimit">
+            <option value="5">5</option>
+            <option value="10">10</option>
+            <option value="25">25</option>
+            <option value="50">50</option>
+          </select><br><br>
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:104]: just checking, these are currently unused right?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,16 +12,24 @@
...
-function addRandomPlayer() {
-  const players =
-      ['Beast', 'Siya Kolisi', 'Latica Nela', 'Chipo Mupeso', 'Gareth Schreuder', 'Andrew James', 'Niaan'];
-  // Pick a random player.
-  const player = players[Math.floor(Math.random() * players.length)];
+async function getMessages() {
+  const response = await fetch('/data');
+  const message = await response.text();
+  document.getElementById('hey-container').innerText = message;
+}
```
Comment (kennzoid) [inline portfolio/src/main/webapp/script.js:22]: Is this currently being used? Or is it just for debugging?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -27,7 +34,38 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jacob!</h1>");
+    Query query = new Query("Task");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList<String> comments = new ArrayList<String>();
```
Comment (wangtong-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: Declare with type "List".

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -27,7 +34,38 @@
...
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("comment");
```
Comment (wangtong-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Please use constant for "comment".

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -27,7 +34,38 @@
...
-}
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = request.getParameter("text-input");
+
+    // Add input to current comments in datastore.
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comment", text);
```
Comment (wangtong-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:63]: Please use constants for these strings.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/index.html:104]: I'll recommit

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 5 above)
Comment (Jaxobc) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:63]: Hey Tong, you'll see the changes in my next pull request

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (Jaxobc) [reply portfolio/src/main/webapp/script.js:22]: Yes, I'll remove it.

**Pull request closed without merging**